docs/introduction/interactions-and-controllers.md

```diff
@@ -499,14 +499,16 @@ For advanced examples on real applications, see the [paint-controls component
 for A-Painter][a-painter] or the [shoot-controls component for
 A-Blast][a-blast].
 
-## Listening for Button and Axis Events
-
-Controllers have many buttons and emit many events. For each button, every time
-a button is pressed down, released, or for some cases, even touched. And for
-each axis (e.g., trackpad, thumbstick), an event is emitted every time it is
-touched. To handle buttons, look for the event name in the respective
-controller component documentation pages at the event tables, then register
-event handlers how we want:
+## Listening for Controller Events
+
+Controllers may have many buttons, many axes, and may emit many events. The
```

Comment:
Thanks. Some alt rewording?

The events emitted by the vendor specific controllers (e.g Quest, Index...) map the lower events emitted by [tracked-controls component]((#tracked-controls-component)) to the specific buttons of each physical controller. To handle input events, we look for the event name...

Reply:
thanks for the suggestion! I think it really helped to break away from the original wording there. I did one more round, happy to keep iterating if something feels off.

Notes on changes to suggestion:
- this is for the A-Frame and vendor specific controllers
- when I see "map" I'm thinking about mapping over an array and it's not necessarily a 1:1 transformation so felt "build on the base level events" captured this a bit better
- "specific buttons" -> "specific events" - not limited to buttons
- "each physical controller" -> "each controller" - I believe this refers to the "controllers" earlier in the sentence which are the software a-frame controllers and may be levels of abstraction away from a physical controller.
- To handle input events -> To handle events - I would imagine controllers don't have to exclusively emit input events. Ex a controller "random-interval" that emits a signal at a configurable random interval to power some dynamic content.

Reply:
Not 100% if talking about `tracked controls` helps the avg user. It should be an implementation detail. Probably more clear to keep it high level. As you mentioned the section [create custom controllers](https://aframe.io/docs/1.2.0/introduction/interactions-and-controllers.html#creating-custom-controllers) exposes lower level detail. Maybe including a link to that section would be enough for those that want to dig deeper?

Reply:
good point, I updated to make it a footnote for the section. will let you decide if it's worth the backlinks, they do link to anchor tags on the same page so perhaps this is trying to do too much and is noisy but I found myself looking for this info when I read this section